src/FSharpVSPowerTools.Logic/LintQuickInfoSource.fs

```diff
@@ -0,0 +1,57 @@
+﻿namespace FSharpVSPowerTools.Linting
+
+open System
+open Microsoft.VisualStudio.Text
+open Microsoft.VisualStudio.Language.Intellisense
+open System.Collections.Generic
+open Microsoft.VisualStudio.Text.Tagging
+open FSharpVSPowerTools.ProjectSystem
+open FSharpVSPowerTools
+open FSharpVSPowerTools.Common
+open System.Reflection
+open System.Windows.Controls
+open System.Windows.Documents
+open System.Windows
+
+type LintQuickInfoSource(buffer: ITextBuffer, tagAggregatorService: IViewTagAggregatorFactoryService) =
+    let mutable disposed = false
+    
+    let createInfoText (tooltips: string list) : UIElement =
+        let textBlock = TextBlock()
+        tooltips |> List.iteri (fun i tooltip ->
+            textBlock.Inlines.Add (Bold (Run "Lint: "))
+            textBlock.Inlines.Add (Run tooltip)
+            if i < tooltips.Length - 1 then
+                textBlock.Inlines.Add (LineBreak()))
+             
+        textBlock.SetResourceReference(TextBlock.BackgroundProperty, VSColors.ToolTipBrushKey)
+        textBlock.SetResourceReference(TextBlock.ForegroundProperty, VSColors.ToolTipTextBrushKey)
+        upcast textBlock
+
+    interface IQuickInfoSource with
+        member __.AugmentQuickInfoSession (session: IQuickInfoSession, quickInfoContent: IList<obj>, 
+                                           applicableToSpan: byref<ITrackingSpan>) = 
+            match session.GetTriggerPoint buffer.CurrentSnapshot |> Option.ofNullable with
+            | None -> ()
+            | Some point ->
+                use tagAggregator = tagAggregatorService.CreateTagAggregator<LintTag> session.TextView
```

Comment:
I think the reason event subscription doesn't work is that the tagger is disposed in this function when `tagAggregator` is out of scope. I think we should create a `tagAggregator` once in the provider and only dispose once it at the end. We did it that way in other features.